src/Compilers/CSharp/Portable/Syntax/StackAllocArrayCreationExpressionSyntax.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.CodeAnalysis.CSharp.Syntax
     public partial class StackAllocArrayCreationExpressionSyntax
     {
         public StackAllocArrayCreationExpressionSyntax Update(SyntaxToken stackAllocKeyword, TypeSyntax type)
-            => Update(StackAllocKeyword, type, Initializer);
+            => Update(stackAllocKeyword, type, Initializer);
```

Comment:
If you wouldn't mind, I think we'd like to have a test case to make sure this doesn't regress in the future.

Reply:
This sounds like even more test syntax generation.

I only found this through IDE0020, which did not appear anywhere else. The existing tests for the auto-generated `Update` functions don't include the hand-written ones.

Adding a test for this specific function doesn't prevent any future regression on another function, where the safest way seems to be even more generated tests.

Reply:
Most of these update functions are generated, and we only handwrite backcompat overloads. We apparently missed adding a test for this one. Please add a test.